packages/run/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@rollup/plugin-run",
+  "version": "1.1.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Run your bundle after you've built it",
+  "license": "MIT",
+  "repository": "rollup/plugins",
+  "author": "Rich Harris",
+  "homepage": "https://github.com/rollup/plugins/packages/run/#readme",
+  "bugs": "https://github.com/rollup/plugins/issues",
+  "main": "index.js",
+  "scripts": {
+    "lint": "pnpm run lint:js && pnpm run lint:docs && pnpm run lint:package",
+    "lint:docs": "prettier --single-quote --write README.md",
+    "lint:js": "eslint --fix --cache *.js",
+    "lint:package": "prettier --write package.json --plugin=prettier-plugin-package",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
The tough part of migrating this plugin is that it needs to have tests written before we can migrate it.

Reply:
@shellscape I was a bit skeptical about making a PR without tests. Should we mock the `child_process` module while writing tests?

If you have some recommendations on how the tests should be, that'd be of help.

Reply:
@shellscape I've added some unit tests for the plugin. I've used `sinonjs` to mock the `child_process` module,  since ava didn't seem to have to an out-of-the-box mocking solution.